doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3677,6 +3677,12 @@ Trusted Firmware-M (TF-M)
 
 The issues in this section are related to the TF-M implementation in the |NCS|.
 
+.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0
+
+NCSDK-22907: TF-M does not build with :kconfig:option:`CONFIG_TFM_ITS_ENCRYPTED` enabled and :kconfig:option:`CONFIG_TFM_PARTITION_PROTECTED_STORAGE` disabled.
+
+  **Workaround:** Enable :kconfig:option:`CONFIG_TFM_PARTITION_PROTECTED_STORAGE` if encrypted ITS is used.
+
 .. rst-class:: v2-4-0
 
 NCSDK-22818: TF-M does not build with :kconfig:option:`CONFIG_TFM_PROFILE_TYPE_MINIMAL` disabled and :kconfig:option:`CONFIG_TFM_LOG_LEVEL_SILENCE` enabled
```

Comment:
Btw, is there a typo in the next line here?

Reply:
honestly not sure what you mean

Reply:
Ok I see, I guess @joerchan just copied the message from the build log, that would look exactly like that

Reply:
error message: \`\`undefined reference to 'end'\`\`